src/tokens/typography.ts

```diff
@@ -1,101 +1,202 @@
-      fontSize: "1.125rem",
-      lineHeight: "1.75rem",
+  title: {
+    lg: {
+      value: {
+        fontFamily: "sans",
+        fontSize: "4xl", 
+        fontWeight: "semibold",
+        lineHeight: "tight",
+        letterSpacing: "balanced",
+      },
     },
-  },
-  xl: {
-    value: {
-      fontSize: "1.25rem",
-      lineHeight: "1.75rem",
+    md: {
+      value: {
+        fontFamily: "sans",
+        fontSize: "3xl", 
+        fontWeight: "semibold",
+        lineHeight: "tight",
+        letterSpacing: "balanced",
+      },
     },
-  },
-  "2xl": {
-    value: {
-      fontSize: "1.5rem",
-      lineHeight: "2rem",
+    sm: {
+      value: {
+        fontFamily: "sans",
+        fontSize: "2xl", 
+        fontWeight: "medium",
+        lineHeight: "tight",
+        letterSpacing: "balanced",
+      },
     },
   },
-  "3xl": {
-    value: {
-      fontSize: "1.875rem",
-      lineHeight: "2.25rem",
+  body: {
+    lg: {
+      value: {
+        fontFamily: "sans",
+        fontSize: "lg", 
+        fontWeight: "normal",
+        lineHeight: "balanced",
+        letterSpacing: "balanced",
+      },
     },
-  },
-  "4xl": {
-    value: {
-      fontSize: "2.25rem",
-      lineHeight: "2.5rem",
+    md: {
+      value: {
+        fontFamily: "sans",
+        fontSize: "md", 
+        fontWeight: "normal",
+        lineHeight: "balanced",
+        letterSpacing: "balanced",
+      },
     },
-  },
-  "5xl": {
-    value: {
-      fontSize: "3rem",
-      lineHeight: "1",
+    sm: {
+      value: {
+        fontFamily: "sans",
+        fontSize: "sm", 
+        fontWeight: "semibold",
+        lineHeight: "balanced",
+        letterSpacing: "balanced",
+      },
     },
   },
-  "6xl": {
-    value: {
-      fontSize: "3.75rem",
-      lineHeight: "1",
+  label: {
+    lg: {
+      value: {
+        fontFamily: "sans",
+        fontSize: "lg", 
+        fontWeight: "medium",
+        lineHeight: "tight",
+        letterSpacing: "balanced",
+      },
+    },
+    md: {
+      value: {
+        fontFamily: "sans",
+        fontSize: "md", 
+        fontWeight: "medium",
+        lineHeight: "tight",
+        letterSpacing: "balanced",
+      },
+    },
+    sm: {
+      value: {
+        fontFamily: "sans",
+        fontSize: "sm", 
+        fontWeight: "medium",
+        lineHeight: "tight",
+        letterSpacing: "balanced",
+      },
     },
   },
-  "7xl": {
+  caption: {
     value: {
-      fontSize: "4.5rem",
-      lineHeight: "1",
+      fontFamily: "sans",
+      fontSize: "xs", 
+      fontWeight: "medium",
+      lineHeight: "tight",
+      letterSpacing: "balanced",
     },
   },
-  "8xl": {
+  code: {
     value: {
-      fontSize: "6rem",
-      lineHeight: "1",
+      fontFamily: "mono",
+      fontSize: "sm", 
+      fontWeight: "medium",
+      lineHeight: "balanced",
+      letterSpacing: "balanced",
     },
   },
-  "9xl": {
-    value: {
-      fontSize: "8rem",
-      lineHeight: "1",
+  heading: {
+    h1: {
```

Comment:
토큰 이름을 피그마와 동일하게 맞춰주시기 바랍니다. 요런 디테일이 사용자 경험에 영향을 줄 수 있거든요.

```suggestion
    "1": {
```